### DISCUSSION TITLE
How to set maxAge?

### DISCUSSION THREAD

**Discussion by luiznasciment0** [arthurfiorette/axios-cache-interceptor#863] [Q&A] [answered]
Hi! I was using `axios-cache-adapter` and now I want to migrate to `axios-cache-interceptor`. 

Here is the code I have to handle `axios-cache-adapter` + `axios`:
```
const axios = require('axios')
const adapter = require('axios-cache-adapter')

const cache = adapter.setupCache({
  maxAge: 2 * 60 * 1000
})

const api = axios.create({
  adapter: cache.adapter
})

...
```

And here is the code I have in order to migrate to `axios-cache-interceptor`:
```
const axios = require('axios')
const { setupCache } = require('axios-cache-interceptor')

const axiosInstance = axios.create()

const api = setupCache(axiosInstance, {  /* here I would like to set maxAge just like I did with `axios-cache-adapter`  */  })
```

Can someone help me? 
Thanks

**Comment by luiznasciment0** [comment]
I was wondering if the `ttl` option can be considered equivalent?

**Comment by luiznasciment0** [comment, marked as answer]
Well, by reading the changelog and [this commit](https://github.com/arthurfiorette/axios-cache-interceptor/commit/be5ee1ea8b4f30c34b68668d9417aef6711bb9a7) I suppose `ttl` is the correct choice in this case. Thanks

**Comment by arthurfiorette** [comment]
yup, they are equivalent
